packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -68,8 +75,13 @@ export const RootScreen = () => {
         <Stack.Navigator
           screenOptions={{ gestureEnabled: false, headerShown: false }}
         >
-          {updateRequired ? (
-            <Stack.Screen name='UpdateStack' component={UpdateRequiredScreen} />
+          {appUpdateRequired || isPendingMandatoryCodePushUpdate ? (
+            <Stack.Screen
+              name='UpdateStack'
+              component={
```

Comment:
Oh that's interesting coming the update screens. I suppose that's fine instead of just having two separate screen definitions